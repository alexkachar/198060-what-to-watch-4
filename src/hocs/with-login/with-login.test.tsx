import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import withLogin from './with-login';
import mockStore from '../../test-data/mock-store';

const MockComponent = () => <div />;
const WrappedComponent = withLogin(MockComponent);

it(`withLogin renders correctly`, () => {
  const tree = renderer
    .create(
        <Provider store={mockStore}>
          <WrappedComponent onLogin={jest.fn()}/>
        </Provider>, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
