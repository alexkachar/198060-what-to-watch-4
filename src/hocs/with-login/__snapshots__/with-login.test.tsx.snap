// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withLogin renders correctly 1`] = `<div />`;
